metricbeat/module/docker/container/data.go

```diff
@@ -31,7 +32,26 @@ func eventMapping(cont *dc.APIContainers) common.MapStr {
 		"status": cont.Status,
 	}
 
+// Check id container have health metrics configured
+if strings.Contains(cont.Status, "(") && strings.Contains(cont.Status, ")") {
+	container, _ := m.dockerClient.InspectContainer(cont.ID)
```

Comment:
One issue I have with this is that for each container this does an additional http request. So if we have 100 containers, this can take quite some time. Is there a possibility to get the data already from the existing data we have? Otherwise we should parallel these requests and make `container.health` a config option so it can be enabled/disabled based on the needs.